app/index.html

```diff
@@ -4,28 +4,51 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2">
     <title>Boats Animator</title>
-    <link rel="stylesheet" href="css/index.css">
+    <link rel="stylesheet" href="css/font-awesome.min.css">
+    <link rel="stylesheet" href="css/fonts.css">
+    <link rel="stylesheet" href="css/animator.css">
```

Comment:
I believe rioforce (or was that me? I forget) created `index.css` because there was conflicting CSS between the two windows and it makes more sense to load just the CSS needed for that window and not everything else. By removing the reference to `index.css` and stuffing everything in `animator.css`, the deliberate separation has been reverted. Please change it back to `index.css` and move the splash-specific CSS style into it.